nodewatcher/plugins/sge.py

```diff
@@ -19,31 +19,24 @@
 
 def hasJobs(hostname):
     # Checking for running jobs on the node
```

Comment:
I'd add a comment in the code with the expected output of the command and a short explanation of the `-g t` option.